.github/workflows/update-github-pages.yaml

```diff
@@ -0,0 +1,29 @@
+name: Deploy github pages
+on:
+    workflow_dispatch:
+    push:
+        branches:
+            - master
+jobs:
+    update-gh-pages:
+        name: Update GitHub pages
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
+            - name: Setup node
+              uses: actions/setup-node@v3
+              with:
+                  node-version: '16'
+                  cache: 'npm'
+            - name: Install dependencies
+              run: npm ci
+            - name: Build mock application
+              run: npm run build:mock
+            - name: Legg til 404 fil
+              run: cp ./.github/pages/404.html ./build/404.html
```

Comment:
This is a copy of the one in portefoljeflate, right? 
This line will probably not work without these pages existing. The code for deploying to gh-pages also exists in main.yaml, so it has to be removed

Reply:
fixed here https://github.com/navikt/veilarbmaofs/pull/309/files#diff-0e246018ec8f1389003192612dc98f3f682b7a74b1cd3d20346a04071d194c4e